src/fr/send.md

```diff
@@ -1 +1,291 @@
+
+- Vous adresser à un utilisateur par son nom
+- Fournir aux utilisateurs un lien précis sur lequel cliquer
+- Envoyer un numéro de transaction unique comme suivi
+- Donner aux utilisateurs une liste dynamique des mesures qu’ils doivent prendre
+
+## Envoyer un courriel
+
+```
+POST /v2/notifications/email
+```
+
+### Corps de la demande
+```json
+{
+  "email_address": "expéditeur@quelquechose.com",
+  "template_id": "f33517ff-2a88-4f6e-b855-c550268ce08a"
+}
+```
+
+### Arguments
+
+**email_address (obligatoire)**
+
+Adresse de courriel du destinataire.
+
+**template_id (obligatoire)**
+
+Pour rechercher l’ID de gabarit :
+
+1. [Connectez-vous à GC Notification](https://notification.canada.ca/sign-in?lang=fr).
+1. Accédez à la page __Gabarits__ et sélectionnez le gabarit approprié.
+1. Sélectionnez __Copier template ID dans le presse-pappier__.
+
+**personnalisation (facultatif)**
+
+Si un gabarit comporte des champs réservés pour des renseignements personnalisés comme le nom ou le numéro de référence, vous devez fournir leurs valeurs dans un dictionnaire avec des paires de valeurs clés. Par exemple :
+
+```json
+"personalisation": {
+  "first_name": "Amala",
+  "application_date": "2018-01-01",
+}
+```
+Vous pouvez ignorer cet argument si un gabarit ne comporte pas de champs réservés pour les renseignements personnalisés.
+
+**référence (facultatif)**
+
+Identificateur que vous pouvez créer au besoin. Cette référence identifie une seule notification ou un lot de notifications. Il ne doit contenir aucun renseignement personnel comme le nom ou l’adresse postale. Par exemple :
+
+```json
+"reference": "STRING"
+```
+Vous pouvez ignorer cet argument si vous n’avez pas de référence.
+
+**email_reply_to_id (facultatif)**
+
+Il s’agit d’une adresse de courriel que vous avez indiquée pour recevoir les réponses de vos utilisateurs. Vous devez ajouter au moins une adresse de courriel de réponse avant que votre service puisse être activé.
+
+Pour ajouter une adresse de courriel de réponse :
+
+1. [Connectez-vous à GC Notification](https://notification.canada.ca/sign-in?lang=fr).
+1. Accédez à la page __Paramètres__.
+1. Dans la section __Courriel__, sélectionnez __Configurer__ à la ligne __Adresses courriel de réponse__.
+1. Sélectionnez __Ajouter une adresse courriel de réponse__.
+1. Entrez l’adresse de courriel que vous souhaitez utiliser, puis sélectionnez __Ajouter__. 
+
+Par exemple :
+
+```json
+"email_reply_to_id": "8e222534-7f05-4972-86e3-17c5d9f894e2"
+```
+
+Vous pouvez ignorer cet argument si votre service n’a qu’une seule adresse de courriel de réponse ou si vous voulez utiliser l’adresse de courriel par défaut.
+
+## Envoyer un fichier par courriel
+
+Pour envoyer un fichier par courriel, ajoutez un espace réservé au gabarit, puis téléchargez un fichier. L’espace réservé contient un lien sécurisé pour télécharger le fichier.
```

Comment:
Should it be "téléverser votre fichier"? (not sure of the word in québécois)

People need to contact us if they want to turn on this feature first at the moment